docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.9"
+services:
+  web:
+    image: "springapp"
+    environment:
+        BOOKSELF_DB_HOST: "172.20.0.3"
```

Comment:
I'm not the best at docker but I think you can just reference `database` here assuming both containers are using the same network (which it looks like they are).

Reply:
I think you're right, but ill merge for now and refactor if this becomes a problem. we may need to fiddle with this docker file in the future anyway